src/features/commands/help.js

```diff
@@ -1,4 +1,5 @@
 const Discord = require('discord.js')
+const server = require('../../index.js')
```

Comment:
It would be better to pull this from the database, it would be best to build `findServerConfig` function, you can reference and copy most things for that from `saveServerConfig` function in https://github.com/Microsoft-community/Dotsimus/blob/5fe0cb172e0013372406b12b26be94abcc3ae1b6/src/db/index.js